README.md

```diff
@@ -86,37 +86,37 @@ Request body:
 ```Accepts: application/json```
 ```json
 {
-  "docker_compose": {
-    "<file_name1>": "<file contents>",
-    "<file_name2>": "<file contents>"
-  }
+ "docker_compose": {
```

Comment:
Are the indentation removals on purpose?

Reply:
yikes! nice catch.  IDE autopilot?  Dunno.

reverted in cd36379